<ui version="4.0" >
 <class>Qt1</class>
 <widget class="QDialog" name="Qt1" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <property name="styleSheet" >
   <string>QPushButton { 
		color: red ;
/*		border-radius : 6px;  */
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>80</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title" >
    <string>GroupBox</string>
   </property>
   <widget class="QRadioButton" name="rbAdd" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>加法</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbSub" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>40</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>减法</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbKk" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>20</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>kk</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbIi" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>40</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>ii</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>150</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="rowCount" >
    <number>0</number>
   </property>
   <property name="columnCount" >
    <number>0</number>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>270</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab1" >
    <attribute name="title" >
     <string>第一页</string>
    </attribute>
    <widget class="QPushButton" name="pbqqqq" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>10</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text" >
      <string>qqqq</string>
     </property>
    </widget>
    <widget class="QLabel" name="image" >
     <property name="geometry" >
      <rect>
       <x>100</x>
       <y>10</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2" >
    <attribute name="title" >
     <string>第二页</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>104</width>
       <height>75</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbqt2" >
     <property name="geometry" >
      <rect>
       <x>130</x>
       <y>30</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text" >
      <string>Qt2</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab3" >
    <attribute name="title" >
     <string>第三页</string>
    </attribute>
    <widget class="QComboBox" name="comboBox" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>78</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text" >
       <string>+</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>-</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pbanniu" >
     <property name="geometry" >
      <rect>
       <x>110</x>
       <y>10</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text" >
      <string>anniu</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>50</y>
       <width>55</width>
       <height>27</height>
      </rect>
     </property>
     <property name="singleStep" >
      <number>2</number>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider" >
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>40</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber" >
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>60</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>78</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLineEdit" name="le1" />
      </item>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le2" />
      </item>
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>=</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leRes" />
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="pbOK" >
        <property name="text" >
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbExit" >
        <property name="text" >
         <string>exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
